### PR TITLE
feat: footer component

### PR DESCRIPTION
**Pull request by JoaquinBattilana**
## General Changes

- Fixes XYZ bug
- Adds XYZ feature
- …

## Developer Notes

Add any notes here that may be helpful for reviewers.

---

## Author Checklist

Please ensure you, the author, have gone through this checklist to ensure there is an efficient workflow for the reviewers.

- [ ]  The base branch is set to `main`
- [ ]  The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
- [ ]  The Github issue has been linked to the PR in the Development section
- [ ]  The General Changes section has been filled out
- [ ]  Developer Notes have been added (optional)

**If the PR is ready for review:**

- [ ]  The PR is in `Open` state and not in `Draft` mode
- [ ]  The `Ready for Dev Review` label has been added

## Reviewer Checklist

Please ensure you, as the reviewer(s), have gone through this checklist to ensure that the code changes are ready to ship safely and to help mitigate any downstream issues that may occur.

- [ ]  End-to-end tests are passing without any errors
- [ ]  Code style generally follows existing patterns
- [ ]  Code changes do not significantly increase the application bundle size
- [ ]  If there are new 3rd-party packages, they do not introduce potential security threats
- [ ]  If there are new environment variables being added, they have been added to the `.env.example` file as well as the pertinant `.github/actions/*` files
- [ ]  There are no CI changes, or they have been approved by the DevOps and Engineering team(s)
- [ ]  Code changes have been quality checked in the ephemeral URL
- [ ]  QA verification has been completed
- [ ]  There are two or more approvals from the core team
- [ ]  Squash and merge has been checked

### CODE REVIEW

**Comment by JoaquinBattilana** [comment]
Link T-8475

**Comment by iamanastasia** [comment]
Hey @JoaquinBattilana great start!

Couple of suggestions:
1. Would be great if links could open in a new tab.
2. Since there are social links in the footer, Github and Discord could be removed from menu.
Screenshot 2023-03-27 at 18 05 20
3. Lens icon is cropped on the right side for some reason. Maybe because of the frame in svg.
Screenshot 2023-03-27 at 18 06 01

And wanted to clarify Docs links:
1. This one is [Docs](https://docs.aave.com/hub/) (docs hub)
2. This one is [Developers](https://docs.aave.com/developers/getting-started/readme) (dev documentation)

**Comment by iamanastasia** [comment]
Hey @JoaquinBattilana great start!

Couple of suggestions:
1. Would be great if links could open in a new tab.
2. Since there are social links in the footer, Github and Discord could be removed from menu.
Screenshot 2023-03-27 at 18 05 20
3. Lens icon is cropped on the right side for some reason. Maybe because of the frame in svg.
Screenshot 2023-03-27 at 18 06 01

And wanted to clarify Docs links:
1. This one is [Docs](https://docs.aave.com/hub/) (docs hub)
2. This one is [Developers](https://docs.aave.com/developers/getting-started/readme) (dev documentation)

File: src/layouts/AppFooter.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+const StyledLink = styled(Link)(({ theme }) => ({
+  color: theme.palette.text.muted,
+  '&:hover': {
+    color: theme.palette.text.primary,
+  },
+  display: 'flex',
+  alignItems: 'center',
+}));
+
+const FOOTER_LINKS = [
```
Comment (foodaka) [inline src/layouts/AppFooter.tsx:17]: @JoaquinBattilana i think they need to be wrapped in a <Trans>

Comment (foodaka) [reply src/layouts/AppFooter.tsx:17]: `<Trans>`

**Comment by foodaka** [comment]
@JoaquinBattilana this one was merged in another PR, closing out nice work!

**Pull request closed without merging**